### PR TITLE
Add shield settings to the default settings provided by govcmslagoon.

### PR DESCRIPTION
**Pull request by steveworley**

### CODE REVIEW

File: .docker/images/govcms8/settings/settings.php [hunk 1]
```diff
@@ -208,6 +208,13 @@ public function __toString() {
 $config['clamav.settings']['scan_mode'] = 1;
 $config['clamav.settings']['mode_executable']['executable_path'] = '/usr/bin/clamscan';
 
+// Enable shield.
+if (getenv('LAGOON') && (getenv('DRUPAL_SHIELD_USER') && getenv('DRUPAL_SHIELD_PASS'))) {
```
Comment (simesy) [inline .docker/images/govcms8/settings/settings.php]: LAGOON locally, where really we should never force shield.

Comment (simesy) [reply .docker/images/govcms8/settings/settings.php]: Unfortunately... I think an empty DRUPAL_SHIELD_USER value is not good enough for shield, which uses an empty switch to disable. So either we set it and lose database control, or it's set to nothing and we can't override the database. We need a different switch to determine if we are applying an override or not...

Comment (simesy) [reply .docker/images/govcms8/settings/settings.php]: Added comments here too https://github.com/govCMS/govcms8-scaffold/pull/15#pullrequestreview-275770002

Comment (simesy) [reply .docker/images/govcms8/settings/settings.php]: I think this should be modified likewise https://github.com/govCMS/govcms8-scaffold-paas/commit/cce7ef0c7139fc5291f97994ea8592b80b06d1df

Comment (srowlands) [reply .docker/images/govcms8/settings/settings.php]: To move to settings.development.php and remove prod tentacles

Comment (ivangrynenko) [reply .docker/images/govcms8/settings/settings.php:212]: Can this PR get some attention? It is an ongoing problem for local development on SaaS.

**Comment by steveworley** [comment]
I think we should revisit this with controlling basic auth at the nginx layer rather than Drupal.

**Pull request closed without merging**